CIP-1.md

```diff
@@ -0,0 +1,19 @@
+Title
+-----
+
+U°Community active key login
+
+U°OS Improvement Proposal
+-------------------------
+
+Add active key registration and authorization.
+
+Users should be able to log in via active key and save key pairs during registration. Brainkey can be used for recovery.
+During the registration process, a password-encrypted json file with account private keys can be saved, and a brainkey is displayed for user to write it down and use later for account keys recovery.
+During the login process a plain text key or an encrypted json can be user for authorization.
```

Comment:
Just a nitpick, pls change "user" to "used" here, I believe this is a typo

Reply:
Fixed